README.rst

```diff
@@ -0,0 +1,66 @@
+Django Zendesk Tickets
+======================
+
+Installation
+------------
+
+At the moment, the only way to install the library is from github
+
+.. code-block:: bash
+
+    pip install git+https://github.com/ministryofjustice/django-zendesk-tickets.git
+
+Settings
+--------
+
+.. code-block:: python
+
+    ZENDESK_BASE_URL=http://zendesk.example.com
+    ZENDESK_API_USERNAME=...
+    ZENDESK_API_TOKEN=...
+    ZENDESK_REQUESTER_ID=...
+    ZENDESK_GROUP_ID=...
```

Comment:
As you can see, some things (e.g. subject, tags, form class) I made per view configurable, but I felt that an app would probably just be submitting tickets for that app. Maybe if you were actually building an application explicitly for interfacing with Zendesk you might want multiple, but in that case I wouldn't use this package.